tutorials/platform/android/android_plugin.rst

```diff
@@ -1,165 +1,388 @@
+    - Or you can switch to the MavenCentral provided dependency::
 
-    - **local**: contains a list of filepaths to the local ``.aar`` binary files the plugin depends on. Similarly to the ``binary`` field (when the ``binary_type`` is ``local``), the local binaries' filepaths can be relative or absolute.
-    - **remote**: contains a list of remote binary gradle dependencies for the plugin.
-    - **custom_maven_repos**: contains a list of URLs specifying the custom maven repositories required for the plugin's dependencies.
+        dependencies {
+            implementation("org.godotengine:godot:4.2.0.stable")
+        }
 
-Loading and using an Android plugin
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+3. After updating the Godot Android library dependency, sync or build the plugin and resolve any compile errors:
+
+    - The ``Godot`` instance provided by ``GodotPlugin::getGodot()`` no longer has access to a ``android.content.Context`` reference. Use ``GodotPlugin::getActivity()`` instead.
+
+4. Delete the ``gdap`` configuration file(s) and follow the instructions in the `Packaging a v2 Android plugin`_ section to set up the plugin configuration.
+
+Packaging a v2 Android plugin
+-----------------------------
+
+As mentioned, a v2 Android plugin is now provided to the Godot Editor as an EditorExportPlugin plugin, so it shares a lot of the `same packaging steps <https://docs.godotengine.org/en/stable/tutorials/plugins/editor/making_plugins.html#creating-a-plugin>`_.
+
+1. Create a folder for the plugin output binaries. The recommended path is ``addons/<plugin_name>/.bin``
+
+2. Create a folder for the `tool scripts <https://docs.godotengine.org/en/stable/tutorials/plugins/editor/making_plugins.html#the-script-file>`_ with the export functionality. The recommended path is ``addons/<plugin_name>/.export``.
+   The created scripts must be ``@tool`` scripts, or else they will not work properly. You need at least two tool scripts:
+
+    - Tool script extending ``EditorExportPlugin`` (e.g: ``addons/<plugin_name>/.export/editor_export_plugin.gd``):
+
+        - This is used to configure the Android plugin and hook it within the Godot Editor's export process. It should look something like this::
+
+            @tool
+            extends EditorExportPlugin
+
+            # Plugin's name
+            var _plugin_name = "<plugin_name>"
+
+            # Specifies which platform is supported by the plugin
+            func _supports_platform(platform):
+                if platform is EditorExportPlatformAndroid:
+                    return true
+                return false
+
+            # Return the paths relative to the 'addons' directory of the plugin's AAR binaries
+            func _get_android_libraries(platform, debug):
+                if debug:
+                    return <paths_to_debug_android_plugin_aar_binaries>
```

Comment:
From the sample project, these seem to need to be `PackedStringArray`s, so it might make sense to show this like:

```suggestion
                    return PackedStringArray(["<paths_to_debug_android_plugin_aar_binaries>"])
```